src/Configuration/ConfigDefinition.php

```diff
@@ -59,8 +59,8 @@ class ConfigDefinition implements ConfigurationInterface
     const PHP_WORKER_NAME = 'name';
 
     /** Default values */
-    const LOG_DIR_DEFAULT = 'var/log';
-    const CACHE_DIR_DEFAULT = 'var/cache/disqontrol';
+    const LOG_DIR_DEFAULT = '/var/log';
```

Comment:
`/var/log` is not writable for most processes, usually a new subdirectory is created for each new application, so `/var/log/disqontrol`?